libs/report/ingestion/src/lib/queries.ts

```diff
@@ -0,0 +1,26 @@
+import { gql } from '@apollo/client/core';
+
+export const TELEMETRY_INITIALIZE_MUTATION = gql`
+  mutation Telemetry($clientName: String!) {
+    telemetry {
+      initialize(input: { context: { client: { id: $clientName } } }) {
+        success
+        oid
+        message
+      }
+    }
+  }
+`;
+
+export const TELEMETRY_REPORT_MUTATION = gql`
+  mutation Report($key: String!, $report: JSON!, $metadata: JSON) {
+    telemetry {
+      report(input: { key: $key, report: $report, metadata: $metadata }) {
+        txId
+        success
+        message
+        diagnostics
+      }
+    }
+  }
+`;
```

Comment:
May be convenient to put hooks here to make these adjustable via env vars. Maybe pre-mature configurability tho

Reply:
I think we can add this in a second iteration when we want to support more files